src/decorators/README.md

```diff
@@ -0,0 +1,9 @@
+# Decorators
+
+This directory contains decorators that can be used in an application.
```

Comment:
Unnecessary statement. Line 7 already makes this clear to those that understand decorators, and line 9 provides instructions for those that do not.